backend/resultRoutes.py

```diff
@@ -35,13 +35,13 @@ def get(self, name, course):
 
 
 class ResultsForEvent(Resource):
-    def get(self, id):
-        return results.getResultsByEvent(id)
+    def get(self, eventID):
+        return results.getResultsByEvent(eventID)
 
 
 class ResultsForCompetitor(Resource):
-    def get(self, id):
-        return results.getResultsByCompetitor(id)
+    def get(self, competitorID):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/brownben/munro/pullRequest?prid=4764451)